mitmproxy/tools/web/web_columns.py

```diff
@@ -0,0 +1,16 @@
+# Auto-generated by generate_web_columns.py
+AVAILABLE_WEB_COLUMNS = [
+    "icon",
+    "index",
+    "method",
+    "version",
+    "path",
+    "quickactions",
+    "size",
+    "status",
+    "time",
+    "timestamp",
+    "tls",
+    "comment",
+]
+DEFAULT_WEB_COLUMNS = ["tls", "icon", "path", "method", "status", "size", "time"]
```

Comment:
This is already hardcoded in mitmproxy/tools/web/webaddons.py, we don't need this to be generated somehow.